apps/pie-microsite/src/index.md

```diff
@@ -1,5 +1,7 @@
 ---
+site_title: PIE
 layout: layouts/page-layout.njk
 title: Principles for Interfaces and Experiences
+description: PIE is the global design system for Just Eat Takeaway, powering all our products.
```

Comment:
Should it be this?
```suggestion
description: PIE is the global design system for Just Eat Takeaway.com, powering all our products.
```

Reply:
I'm just going by the copy on the designs. I'll check with the designers

Reply:
updated! thanks for catching this